confgenerator/logging_receivers.go

```diff
@@ -180,7 +181,7 @@ func (r LoggingReceiverTCP) Components(tag string) []fluentbit.Component {
 			"Tag":    tag,
 			"Listen": r.ListenHost,
 			"Port":   fmt.Sprintf("%d", r.ListenPort),
-			"Format": "json",
```

Comment:
I would keep this hardcoded as it was before, because we aren't necessarily using the same strings/config format as fluentbit. The rest of this config is all still hardcoded for streaming JSON.

Reply:
(My fear is that someone will see your change and think "great, we already planned ahead for more formats, I don't need to change anything to support the next format")